client/performance-profiler/components/metrics-insight/insight-header.tsx

```diff
@@ -27,10 +29,13 @@ export const InsightHeader: React.FC< InsightHeaderProps > = ( props ) => {
 			>
 				{ title }
 			</Markdown>
-			{ value && isMobile() && <span className="value is-mobile"> { value }</span> }
+			{ value && isMobile() && (
+				<span className={ clsx( 'value is-mobile', { [ type ]: true } ) }> { value }</span>
+			) }
 			{ value && ! isMobile() && (
 				<span>
-					&nbsp;&minus;&nbsp;<span className="value"> { value }</span>
+					&nbsp;&minus;&nbsp;
+					<span className={ clsx( 'value', { [ type ]: true } ) }> { value }</span>
```

Comment:
`{ [ type ]: true }` -> mind blowing to me, very nice :D

Reply:
Yeah, sorry about it, this is how clsx handles the object. :D

Reply:
Nah, I liked it, I just never thought on using a property value with `true`. I would probably do `{  warning: type === 'warning', fail: type === 'fail' }` but I liked yours more.